multimodal/src/autogluon/multimodal/utils.py

```diff
@@ -901,25 +902,24 @@ def save_pretrained_models(
         model = model.model
     for per_model in model:
         if per_model.prefix.lower().startswith((CLIP, HF_TEXT)):
-            per_model.model.save_pretrained(os.path.join(path, per_model.prefix))
+            per_model.config.save_pretrained(os.path.join(path, per_model.prefix))
```

Comment:
For huggingface models, you may also need to save the tokenizer.

Reply:
The text tokenizers are saved by `save_text_tokenizers()`.